week-3/B-array-some/exercise.js

```diff
@@ -15,6 +15,13 @@ var pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
 var students = ["Islam", "Lesley", "Harun", "Rukmini"];
 var mentors = ["Daniel", "Irina", "Mozafar", "Luke"];
 
+function isNull(value) {
+  if (value === null) {
+    process.exit(1);
+  }
+}
+pairsByIndex.some(isNull);
```

Comment:
I think this got a little mixed up - `some()` will return you a boolean. If you use `value === null` in a function that you put into `some()` - that will check every **value** in the `pairsByIndex` array - and if any of them is `null`, it'll return `true`. Then you can use that boolean in a condition, and only if it's `true` - call `process.exit(1);`

Hope this makes more sense now - can you please try again? :)